roles/ealapps/files/eal_app

```diff
@@ -0,0 +1,26 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
What does this private key do?  And can we add a `.key` to the filename?

Reply:
allows us to clone a private repo.